packages/ui/src/forum/components/PostList/PostListItem.tsx

```diff
@@ -76,24 +78,19 @@ export const PostListItem = forwardRef<HTMLDivElement, PostProps>(
           )}
         </MessageBody>
         <ForumPostRow>
+          <ButtonsGroup>{reaction && <LikeButton counter={reaction.length} />}</ButtonsGroup>
           {isThreadActive && !editing && (
             <ButtonsGroup>
-              {reaction && (
-                <Button size="small">
-                  <HeartIcon />
-                  {!!reaction.length && reaction.length}
-                </Button>
-              )}
```

Comment:
I think the like button should be hidden when the thread is not active (see #1224)